lib/puppet/application/cert.rb

```diff
@@ -161,26 +65,12 @@ def help
 HELP
   end
 
-  def main
-    help
-  end
-
   def setup
     deprecate
   end
 
   def parse_options
-    # handle the bareword subcommand pattern.
-    result = super
-    unless self.subcommand then
-      if sub = self.command_line.args.shift then
-        self.subcommand = sub
-      else
-        puts help
-        exit
-      end
-    end
-
-    result
+    puts help
```

Comment:
Should this be `Puppet.warn` or something similar (or have we not gotten to where Puppet is initialized yet?)

Reply:
I'll try it. There was a rescue block here previously that used "puts" so I went with that. But something through the logger would probably be better.

Reply:
When I went through `Puppet.warning` or `Puppet.err`, nothing was emitted. Logging must not be set up right at that point. Going to leave it as `puts`.